CSSOM updates to match web API templates (#44993)
mdn/content: files/en-us/web/api/css/index.md

{{APIRef("CSSOM")}}

The **`CSS`** interface of the [CSS Object Model (CSSOM)](/en-US/docs/Web/API/CSS_Object_Model) groups useful CSS-related static methods.

> No objects with this interface are implemented: it contains only static methods and is therefore a utilitarian interface.

Reviewer: A utilitarian interface?

```suggestion
This interface contains only static methods and cannot be instantiated.
```
